client/src/App.js

```diff
@@ -3,6 +3,7 @@ import { useSelector, useDispatch } from 'react-redux';
 import LoginButton from './components/Login/LoginButton';
 import { checkUser } from './redux/features/auth/authSlice';
 import 'antd/dist/antd.css';
+import ProfileTabs from './components/ProfileTabs/ProfileTabs';
```

Comment:
Since you are already inside a ProfileTabs folder, then just name your files index.jsx and style.css